week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function setAlarm() {
+  let count = document.getElementById("timeRemaining");
+  let setIt = document.getElementById("set");
+  let pause = document.getElementById("p");
+  let Restart = document.getElementById("re");
+  let Counter;
+  pause.addEventListener("click", function () {
+    clearInterval(Counter);
+  });
+  Restart.addEventListener("click", function () {
+    Counter = setInterval(CountDown, 1000);
+  });
+  let TheTime;
+  setIt.addEventListener("click", function () {
+    TheTime = document.getElementById("alarmSet").value;
+
+    if (TheTime <= 60) {
+      count.innerHTML = "Time Remaining: 00:" + TheTime;
+    }
+  });
```

Comment:
This should also start the countdown. It only works if you double click on the Set button which is a bit dodgy.